.github/workflows/smoke.yml

```diff
@@ -53,8 +65,21 @@ jobs:
       run: |
         make go-install
 
-    - name: Smoke Test
+    - name: Update microk8s operator image
+      if: matrix.cloud == 'microk8s'
+      run: |
+        make microk8s-operator-update
```

Comment:
Would be better to use a fake docker account to ensure we are testing the locally built image.